src/stores/game.js

```diff
@@ -462,7 +462,7 @@ export const useGameStore = defineStore('game', {
     },
     async requestSpectateLeave() {
       return new Promise((resolve, reject) => {
-        io.socket.get('/api/game/spectateLeave', (res, jwres) => {
+        io.socket.post(`/api/game/${this.id}/spectate/leave`, (_res, jwres) => {
```

Comment:
This was the cause of the error that was being logged and therefore crashing the app. Url for this endpoint was switched in #1135